awx/main/tasks.py

```diff
@@ -1114,6 +1121,14 @@ def event_handler(self, event_data):
                 event_data.pop('parent_uuid', None)
         if self.parent_workflow_job_id:
             event_data['workflow_job_id'] = self.parent_workflow_job_id
+        if self.host_map:
+            host = event_data.get('event_data', {}).get('host', '').strip()
+            if host:
+                event_data['host_name'] = host
```

Comment:
I'm a little confused by the addition of `host_name`. We already had this information in the event data from Ansible. If we duplicate it, I worry that it will make the (already fairly intractable) event data even harder to follow.

Reply:
`host_name` is a column on `main_jobevent` that we're populating with this code (like `event` or `stdout`).

This doesn't show up duplicated in the *Ansible* event data structure.  Here's what an actual event looks like after this runs:

![image](https://user-images.githubusercontent.com/214912/72188637-ca603580-33c8-11ea-8e9a-86277e0967f6.png)

Reply:
So this assignment of host_name used to live at

https://github.com/ansible/awx/pull/5618/files#diff-9b1e5b80fcb01fab3e7a0ece4e371a50L736

and it is moved here, to before the data gets shipped

Reply:
Yep, exactly.  Under this model, the event *going into RabbitMQ* already has `host_id` and `host_name` set, and they just get mapped *directly* to columns on e.g., `main_jobevent`.